.eslintrc.js

```diff
@@ -152,6 +152,7 @@ module.exports = {
         'vars-on-top': 'off',
         yoda: 'off',
         'prettier/prettier': 'off',
+        'import/order': 'off',
```

Comment:
Looking at this PR: I'm gonna remove this commit and fix the offending lines instead!